README.md

```diff
@@ -13,7 +13,7 @@
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://mudasiri.github.io/portfolio/)
+[Live Demo Link](#)
```

Comment:
Kindly deploy your project using GitHub pages and please include the link in your README file. This is a project requirement. :+1: :smiley: :smiley: 
![image](https://user-images.githubusercontent.com/98363075/196595799-bc5173f9-5e9c-4ef0-b52d-fbf7b4214587.png)